### PR TITLE
Add sticky note styling for Question Wrappers

### PR DESCRIPTION
**Pull request by sfentress**

### CODE REVIEW

File: src/components/question-wrapper.tsx [hunk 1]
```diff
@@ -52,33 +52,40 @@ export default class QuestionWrapper extends React.Component<IProps, IState> {
...
+      visibleText = distractorsExplanation;
       wrappedContentClass += " " + css.distractors;
     } else if (activeTab === "TeacherTip") {
-      footer = teacherTip;
+      visibleText = teacherTip;
       wrappedContentClass += " " + css.teacherTip;
     } else if (activeTab === "Exemplar") {
-      footer = exemplar;
+      visibleText = exemplar;
       wrappedContentClass += " " + css.examplar;
```
Comment (DALove1025) [inline src/components/question-wrapper.tsx:78]: I hadn't noticed this before, but the spelling of "examplar" is both odd and inconsistent. It occurs here as well as in it's definition in the question-wrapper.sass file, around line 72. Prolly not worth fixing, but it jumped out at me.

Attach files by dragging & dropping, selecting or pasting them.

**Review by DALove1025 — APPROVED** [review]
Looks great. I did notice one thing during testing (prolly not worth addressing) that I found a bit odd. In the authoring preview of the question wrapper, using a sticky note on a distractor looks like this:

image

But, when I click on the  TeacherTip tab, the sticky has a darker-blue top-boarder....

image

Which may well be the intended behavior. But, isolated, like that in the authoring preview, it looks a little funny to me that the sticky in TeacherTip tab looks different than the sticking in the Correct, Distractors, or Exemplar tabs.

File: src/components/question-wrapper.tsx (hunk 1 above)
Comment (sfentress) [reply src/components/question-wrapper.tsx:78]: Ah yes, it looks like there was a typo in the css file.

**Comment by sfentress** [comment]
I mentioned leaving that behavior in the commit message -- the idea being that sticky notes appear to be intended only for Teacher Tips when wrapping interactives, which are really a different use-case from the correct/incorrect tips for multiple-choice questions, and yet we're not differentiating them in authoring.

I figured if I put the bars on (which haven't been styled by Michael, because it's not an expected use-case, although we could guess the colors) no one would think twice about this issue, but leaving them off (1) is a hint that authors shouldn't make correct/incorrect sticky notes, and (2) will force us to discuss this, and maybe we'll decide we should separate out the two uses of this wrapper.

(Likewise, the image overlay doesn't make sense when wrapping multiple-choice questions.)

**Pull request merged**